ui/admin/app/components/form/host-set/azure/index.hbs

```diff
@@ -69,96 +69,34 @@
     </F.HelperText>
   </InfoField>
 
-  <Rose::Form::Fieldset
-    @helperText={{t 'resources.host-set.form.preferred_endpoints.help'}}
-    @linkText={{t 'actions.learn-more'}}
-    @link={{doc-url 'host-set.preferred-endpoints'}}
+  <Form::Field::ListWrapper
+    @layout='horizontal'
+    @isOptional={{true}}
+    @name='preferred_endpoints'
+    @options={{@model.preferred_endpoints}}
+    @addRow={{fn @addRow 'preferred_endpoints'}}
+    @removeOptionByIndex={{fn @removeOptionByIndex 'preferred_endpoints'}}
+    @disabled={{form.disabled}}
+    @hasSingleTextInputField={{true}}
```

Comment:
None blocking question: What is the purpose of declaring hasSingleTextInputField? it is for having just 1 input file to add options to the list?